db/helpers/birthday_helper.py

```diff
@@ -86,3 +109,54 @@ def get_todays_birthdays() -> List[Birthday]:
     for birthday in queried_birthdays:
         birthdays[birthday.guild_id].append(birthday)
     return birthdays
+
+def is_last_day_of_month() -> bool:
+    now = dt.datetime.utcnow()
+    return now.day == calendar.monthrange(now.year, now.month)[1]
+
+def get_todays_baby_month_milestones() -> List[Birthday]:
+    """
+    Gets the babies born on this day or the month for the last 12 months
+    :return: array of guild_ids with an array of Birthdays
+    """
+    last_year = dt.datetime.utcnow().replace(tzinfo=dt.timezone.utc) - dt.timedelta(days=365)
+    now = dt.datetime.utcnow().replace(tzinfo=dt.timezone.utc)
+    queried_birthdays = DB.s.execute(
+        sa.text(f"""
+        SELECT *
+        FROM birthdays
+        WHERE day = {now.day} and ((year = {now.year} and month < {now.month}) or (year = {last_year.year} and month >= {now.month}))
+        ORDER BY guild_id desc;
+        """)
+    ).all()
+    last_day_of_month = is_last_day_of_month()
+    if last_day_of_month:
```

Comment:
If it's the last day of the month (and that isn't the 31st) we need to get all the birthdays between the last day and the 31st. Example, for Feb 29th, we should also grab all the birthdays on the 30th and 31st.